### PR TITLE
Feat/todo

### PR DESCRIPTION
**Pull request by viru2001**
add Todo list

### CODE REVIEW

**Review by rahul31agrawal — APPROVED** [review]
Hi Viresh,

1) UI looks awsome
2) App is working smoothly
3) The code structure & naming convention is good

Optional suggestion:- 
1) The Time & User name should  be displayed at the center of the page
2) the Todo button is at the corner of the page, so it is  not visible properly, so i suggest it should be placed where it is easily visible to user.
Also the todo task is visible only on hover, i suggest it should be fixed after addition of the task

Rest everything looks good to merge

Thanks

File: public/manifest.json [hunk 1]
```diff
@@ -1,25 +1,9 @@
...
-      "type": "image/png",
-      "sizes": "512x512"
-    }
-  ],
-  "start_url": ".",
-  "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
+  "short_name": "Plan Your Day",
+  "manifest_version": 2,
```
Comment (irajbhinde) [inline public/manifest.json]: make it version 3, 2 is going to be deprecated

File: src/components/TodoButton/TodoButton.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        onClose={handleClose}
+        anchorOrigin={{
+          vertical: "top",
+          horizontal: "right",
+        }}
+        transformOrigin={{
+          vertical: "bottom",
+          horizontal: "right",
+        }}
+        // sx={{ width: "600" }}
```
Comment (irajbhinde) [inline src/components/TodoButton/TodoButton.jsx]: remove commented

**Review by irajbhinde — COMMENTED** [review]
Add an onkeypress enter for name, creating a new todo & editing the todo.

Rest of the things look great! Kudos

**Pull request merged**